### PR TITLE
Analyzer no longer available. Not supported for 2016/2016

### PR DESCRIPTION
**Pull request by cabaudri**
The best practices analyzer was only in a beta stage in 2013, the actual link no longer exists http://bestpracticesanalyzer.microsoft.com/clickonce/setup.exe 

The only one will be: 
https://techcommunity.microsoft.com/t5/exchange-team-blog/beta-of-microsoft-office-365-best-practices-analyzer-for/ba-p/591294

Tools is no longer available in Exchange 2016 or 2019.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/about-the-office-365-best-practices-analyzer-for-exchange-server.md [hunk 1]
```diff
@@ -27,23 +27,23 @@ description: "Admins can learn about the requirements and capabilities of the BP
...
 
-The Office 365 Best Practices Analyzer is an automated tool that evaluates the health and readiness of your on-premises Exchange environment. You can run the Office 365 Best Practices Analyzer on your Exchange server at any time to assess the state of your Exchange configuration.
+The Office 365 Best Practices Analyzer is an automated tool that evaluates the health and readiness of your on-premises Exchange environment. You can run the Office 365 Best Practices Analyzer on your Exchange Server 2013 at any time to assess the state of your Exchange configuration.
 
 ## Overview
 
 You can use the Office 365 Best Practices Analyzer in the following environments:
 
-- On-premises Exchange server only (Exchange 2013 or later)
+- On-premises Exchange server only (Exchange 2013)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/about-the-office-365-best-practices-analyzer-for-exchange-server.md]:
```suggestion
- On-premises Exchange server only (Exchange Server 2013)
```

File: Exchange/ExchangeHybrid/about-the-office-365-best-practices-analyzer-for-exchange-server.md [hunk 2]
```diff
@@ -27,23 +27,23 @@ description: "Admins can learn about the requirements and capabilities of the BP
...
 
-- Hybrid configuration (with Exchange 2013 or later)
+- Hybrid configuration (with Exchange 2013)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/about-the-office-365-best-practices-analyzer-for-exchange-server.md]:
```suggestion
- Hybrid configuration (with Exchange Server 2013)
```

File: Exchange/ExchangeHybrid/about-the-office-365-best-practices-analyzer-for-exchange-server.md [hunk 3]
```diff
@@ -64,4 +64,4 @@ Step 4: Learn more about any warnings or failures.|1. On the **Best practices sc
 
 ## Next steps
 
-After fixing any reported problems, you can run the Office 365 Best Practices Analyzer again from your Exchange server.
+After fixing any reported problems, you can run the Office 365 Best Practices Analyzer again from your Exchange server 2013.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/about-the-office-365-best-practices-analyzer-for-exchange-server.md]:
```suggestion
After fixing any reported problems, you can run the Office 365 Best Practices Analyzer again from your Exchange Server 2013.
```

**Comment by mattpennathe3rd** [comment]
@JohanFreelancer9 @scanum No further work on this issue, please. This tool has been retired and the content is to be removed. I will handle.

**Pull request closed without merging**